### ISSUE TITLE
NVDA doesn't announce matTooltip in IE 11

### ISSUE THREAD

**Issue by diminutivesloop** [angular/components#12298]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
matTooltip text should be announced by screen reader when associated control is focused.

#### What is the current behavior?
In IE 11 matTooltip text is not read by NVDA.

#### What are the steps to reproduce?
https://stackblitz.com/edit/angular-5axarj

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular 6.0.9
Material 6.4.0
Browser: IE 11
Screen Reader: NVDA 2017.4

**Comment by crisbeto** [comment]
I can reproduce the issue as well. @jelbourn by the looks of it, the issue is with the `AriaDescriber` which has `display: none` and `aria-hidden` that cause IE not to read out the messages ([for reference](https://developer.paciellogroup.com/blog/2014/06/aria-labelledby-aria-describedby-support-popular-windows-browsers/)). We can hide the element using `cdk-visually-hidden`, but if we were to remove the `aria-hidden`, the messages will now become part of the page, which means that they'll be read out when the user uses the arrow keys. I don't see a nice way around it, aside from only adding `aria-hidden` on non-MS browsers.

**Comment by jelbourn** [comment]
What happens if the parent element has `aria-hidden="true"` and the child description has `aria-hidden="false"`?

**Comment by crisbeto** [comment]
The message doesn't get read out in this case either.

**Comment by jelbourn** [comment]
Yeah, this seems like a case where it makes sense to do something different for IE11 then.

**Comment by diminutivesloop** [comment]
Any idea when a fix for this will be merged?
